File: app/src/main/java/com/rifafauzi/moviecatalogue/model/Movies.java
package com.rifafauzi.moviecatalogue.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Movies {

    @SerializedName("poster_path")
    @Expose
    private String posterPath;

    @SerializedName("overview")
    @Expose
    private String overview;

    @SerializedName("release_date")
    @Expose
    private String releaseDate;

    @SerializedName("id")
    @Expose
    private int id;

    @SerializedName("title")
    @Expose
    private String title;

    public Movies(String posterPath, String overview, String releaseDate, int id, String title) {
        this.posterPath = posterPath;
        this.overview = overview;
        this.releaseDate = releaseDate;
        this.id = id;
        this.title = title;
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public String getOverview() {
        return overview;
    }

    public String getReleaseDate() {
        return releaseDate;
    }
}

File: app/src/main/java/com/rifafauzi/moviecatalogue/model/Movies.kt
package com.rifafauzi.moviecatalogue.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Movies(@field:SerializedName("poster_path")
             @field:Expose
             val posterPath: String,

             @field:SerializedName("overview")
             @field:Expose
             val overview: String,

             @field:SerializedName("release_date")
             @field:Expose
             val releaseDate: String,

             @field:SerializedName("first_air_date")
             @field:Expose
             val firstAirDate: String,

             @field:SerializedName("id")
             @field:Expose
             val id: Int,

             @field:SerializedName("name")
             @field:Expose
             val name: String,

             @field:SerializedName("title")
             @field:Expose
             val title: String)